core/quill.js

```diff
@@ -72,8 +72,9 @@ class Quill {
     this.clipboard = this.theme.addModule('clipboard');
     this.history = this.theme.addModule('history');
     this.theme.init();
-    let contents = this.clipboard.convert(`<div class='ql-editor' style="white-space: normal;">${html}<p><br></p></div>`);
-    this.setContents(contents);
```

Comment:
I think pasteHTML below needs to be removed instead of this. pasteHTML is deprecated.